webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5441,61 +5445,67 @@ msgstr "Religion"
-"Freischaltcode finden Sie auf der letzten Seite des Briefes."
-
-#: app/templates/unlock_code/registration_success.html:18
-msgid "register.success.preparation.heading"
-msgstr "Wie Sie sich auf Ihre Steuererklärung vorbereiten können"
-
-#: app/templates/unlock_code/registration_success.html:19
-msgid "register.success.preparation.intro"
-msgstr ""
-"Unsere Vorbereitungshilfe gibt Ihnen einen Überblick darüber, welche "
-"Unterlagen Sie für die Erstellung Ihrer Steuererklärung beim Steuerlotsen"
-" brauchen, und erklärt, wie Sie nach Erhalt des Briefes fortfahren."
-
-#: app/templates/unlock_code/registration_success.html:21
-msgid "register.success.preparation.button"
-msgstr "Vorbereitungshilfe herunterladen"
+#~ msgid "form.unlock-code-revocation.header-title"
+#~ msgstr "Stornieren - Der Steuerlotse für Rente und Pension"
+
+#~ msgid "register.success.next-steps.heading"
+#~ msgstr "So geht es weiter"
+
+#~ msgid "register.success.next-steps.step-1"
+#~ msgstr ""
+#~ "Sie bekommen von Ihrem Finanzamt den "
+#~ "<b>Brief mit Ihrem persönlichen "
+#~ "Freischaltcode</b> zugeschickt."
+
+#~ msgid "register.success.next-steps.step-2"
+#~ msgstr ""
+#~ "Sie können sich auf Ihre Steuererklärung"
+#~ " vorbereiten bis Sie den Brief "
+#~ "erhalten haben. Sammeln Sie dazu alle"
+#~ " notwendigen Unterlagen sowie Belege. Eine"
+#~ " Übersicht über die notwendigen Unterlagen,"
+#~ " die Sie für die Erstellung Ihrer "
+#~ "Steuererklärung brauchen, finden Sie in "
+#~ "unserer <a href=%(link)s>Vorbereitungshilfe</a>."
+
+#~ msgid "register.success.next-steps.step-3"
+#~ msgstr ""
+#~ "Wenn Sie den Brief erhalten haben "
+#~ "und vorbereitet sind, gehen Sie erneut"
+#~ " auf www.steuerlotse-rente.de"
+
+#~ msgid "register.success.next-steps.step-4"
+#~ msgstr ""
+#~ "Wählen Sie den Menüpunkt <a "
+#~ "href=%(link)s> Ihre Steuererklärung</a> und "
+#~ "melden sich mit Ihrem Freischaltcode an."
+#~ " Nach der Anmeldung können Sie das"
+#~ " Steuerformular ausfüllen und verschicken."
+
+#~ msgid "register.success.letter.heading"
+#~ msgstr "So sieht der Brief aus, den Sie erhalten werden"
+
+#~ msgid "register.success.letter.intro"
+#~ msgstr ""
+#~ "Auf dem Brief wird der "
+#~ "DigitalService4Germany als Antragsteller angegeben."
+#~ " Die Organisation ist der Betreiber "
+#~ "des Steuerlotsen. Ihren Freischaltcode finden"
+#~ " Sie auf der letzten Seite des "
+#~ "Briefes."
+
+#~ msgid "register.success.preparation.heading"
+#~ msgstr "Wie Sie sich auf Ihre Steuererklärung vorbereiten können"
+
+#~ msgid "register.success.preparation.intro"
+#~ msgstr ""
+#~ "Unsere Vorbereitungshilfe gibt Ihnen einen "
+#~ "Überblick darüber, welche Unterlagen Sie "
+#~ "für die Erstellung Ihrer Steuererklärung "
+#~ "beim Steuerlotsen brauchen, und erklärt, "
+#~ "wie Sie nach Erhalt des Briefes "
+#~ "fortfahren."
+
+#~ msgid "register.success.preparation.button"
+#~ msgstr "Vorbereitungshilfe herunterladen"
```

Comment:
I know that this was not because of this PR. However, we could delete those messages now because they are part of the frontend.

Reply:
I'm fixing this in Victors PR